tests/managed/methods.cs

```diff
@@ -88,6 +88,12 @@ public void Remove (Item item)
 		}
 	}
 
+	public class DuplicateMethods {
+		public int DoIt () { return 42; }
+		public int DoIt (int i) { return 42; }
+		public int DoIt (string s) { return 42; }	
```

Comment:
@spouliot 
`		public int DoIt (int i, int i) { return 84; }
`
gives
`/Users/donblas/Programming/Embeddinator-4000/tests/managed/methods.cs(31,31): Error CS0100: The parameter name 'i' is a duplicate (CS0100) (managed)`